tests/transformers/test_modeling.py

```diff
@@ -12,13 +12,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import unittest
-from paddlenlp.transformers import TinyBertModel
+import shutil
+from tempfile import TemporaryDirectory
+from tests.testing_utils import slow
+from multiprocessing import Pool
+from paddlenlp.transformers import TinyBertModel, BertModel
+from paddlenlp.utils.env import MODEL_HOME
+
+
+def download_bert_model(model_name: str):
+    """set the global method: multiprocessing can not pickle local method
+
+    Args:
+        model_name (str): the model name
+    """
+
+    model = BertModel.from_pretrained(model_name)
+    # free the model resource
+    del model
 
 
 class TestModeling(unittest.TestCase):
     """Test PretrainedModel single time, not in Transformer models"""
 
+    @slow
     def test_from_pretrained_with_load_as_state_np_params(self):
         """init model with `load_state_as_np` params"""
         TinyBertModel.from_pretrained("tinybert-4l-312d", load_state_as_np=True)
+
+    @slow
+    def test_multiprocess_downloading(self):
+        num_process, num_iter = 10, 20
+        small_model_path = "https://paddlenlp.bj.bcebos.com/models/community/__small_models__/bert-base-uncased/model_state.pdparams"
+
+        from paddlenlp.transformers.model_utils import get_path_from_url
```

Comment:
maybe just import at the top? we will re-use to test other stuff in `model_utils.py` so we will import everything anyway